src/org/traccar/protocol/MeiligaoProtocolDecoder.java

```diff
@@ -151,6 +151,19 @@ private String getMeiligaoServer(Channel channel) {
         return server;
     }
 
+    private String decodeAlarm(short value) {
+        switch (value) {
+        case 0x01: return Position.ALARM_SOS;
```

Comment:
I have a question, what is the right indentation for `case` ?
Eclipse wants the same level as `switch` but for example in 'OsmAndProtocolDecoder` it is on 4 spaces right.